datasets/AlMadi2018/stimulus_1706913863030379050/trial_0_participant_1_1706913863030379050.json

```diff
@@ -1,5 +1,37 @@
 {
+    "time_stamps": [
+        178149,
+        178253,
+        178279,
+        178398,
+        178428,
+        178553,
+        178574,
+        178661,
+        178685,
+        178852,
+        178877,
+        179063,
+        179090,
+        179309,
+        179336,
+        179569,
+        179585,
+        179715,
+        179745,
+        179991,
+        180014,
+        180139,
+        180270,
+        180385,
```

Comment:
180385 is the time_stamp of a saccade, so there's an error in the calculation of all the files presumably.

To fix this problem, start by selecting only fixations from the eye events, then take the timestamps associated with each fixation.  Let me know if you have any questions, and thanks again for working on this.

Reply:
Done!